src/coreclr/interpreter/compiler.cpp

```diff
@@ -3940,19 +3940,15 @@ void InterpCompiler::EmitCall(CORINFO_RESOLVED_TOKEN* pConstrainedToken, bool re
     bool isDelegateInvoke = false;
     bool isJmp = (*m_ip == CEE_JMP);
 
-    CORINFO_RESOLVED_TOKEN resolvedCallToken;
-    CORINFO_CALL_INFO callInfo;
+    CORINFO_RESOLVED_TOKEN resolvedCallToken{};
+    CORINFO_CALL_INFO callInfo{};
```

Comment:
`CORINFO_CALL_INFO` is sizable struct. IIRC, zero initializations like this showed up in the profiles in the past.